Packs/AzureLogAnalytics/ReleaseNotes/1_1_49.md

```diff
@@ -4,4 +4,4 @@
 ##### Azure Log Analytics
 - Updated the Docker image to: *demisto/crypto:1.0.0.3539024*.
 
-- Updated the Azure Log Analytics integration to be more efficient with the credential input.
+- Fixed an issue where reconfiguring an integration instance would not properly update to a new API token, instead retaining an old one.
```

Comment:
@Moish-Gilboa In my suggestion for release notes I wrote API token cause it was right for that integration, need to adjust the RN to match the relevant param (API key, toke, password, certificate, etc...)